docs/faq.rst

```diff
@@ -398,6 +398,52 @@ Two mailing lists.
 * `scylladb-dev <https://groups.google.com/d/forum/scylladb-dev>`_: Discuss the development of Scylla itself.
 * `scylladb-users <https://groups.google.com/d/forum/scylladb-users>`_: Discuss using Scylla and developing client applications.
 
+What is Eventual Consistency, and how is it different from Strong Consistency?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To understand the difference, I’ll start with defining `consistency <https://www.scylladb.com/glossary/database-consistency/>`_. 
```

Comment:
@nyh , thanks for the review. 
I was considering adding this directly to the community forum, but @tzach suggested the docs would be a better place for it, and I agree. 
@annastuchlik is responsible for making sure it's up to date. Since it's open source, anyone can open an issue or send a PR if something isn't. 
Regarding interlinking, AFAIK, it is currently possible to interlink, as each question has a "permalink" to it. @annastuchlik correct me if I'm wrong.
Regarding searchability, while the questions do have a direct link, a quick test shows that specific questions are indexed, both in Google and in the internal search, however, the resulting link is to the general FAQ page and not to the question that was searched for. 
@annastuchlik any thoughts on this? If this is indeed the case, it would make sense to change the FAQ (and other pages) to have individual pages for each question, as @nyh suggests.

Reply:
> Regarding interlinking, AFAIK, it is currently possible to interlink, as each question has a "permalink" to it. @annastuchlik correct me if I'm wrong.

Even if this is true that links into the middle of the FAQ work, the question is - what the point of having a FAQ instead of individual documents on individual topics, with links between related topics?

I remember 30 years ago, I used to find a FAQ on a subject on some FTP site, print it out (on paper!) and read it cover to cover. I don't think anyone does this any more - but maybe the thinking is that some people still do it?
One of the consequences of bundling questions together as a FAQ also is tradeoff for answer length: If you expect people to read the FAQ entirely (like I used to do in the past), each answer must be short - and your very detailed answer about consistency is perhaps too long. But if you expect people to read the answer of just the one question, there is almost no limit to the length of the answer. It can even start short and then proceed to give more details.

Reply:
My take is that people read a specific question and answer. Let's see how @annastuchlik sees it.